matlab/matlab-client/src/matlab/PAsolve.m

```diff
@@ -759,6 +780,13 @@ function initSolveConfig(solve_config,opt,sched)
 % Initialize Task Config Input Parameters
 function [input,main,taskFilesToClean,outVarFiles]=initParameters(solveid,NN,MM,Tasks,opt,taskFilesToClean,task_config,allfuncs,pa_dir,subdir,fs)
 
+    function answer = topath(list)
+        answer = list{1};
+        for z = 2:length(list)
```

Comment:
the index of arrays or list starts at 1. Here we need one minimum element in the path built and append each subsequent element with a semicolon